.github/workflows/deploy.yml

```
@@ -7,20 +7,27 @@ on:
 jobs:
   test:
     uses: "ambanum/OpenTermsArchive/.github/workflows/test.yml@main"
+
+  get_instances:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set matrix with instance names
+        id: get_instance_names
+        uses: mikefarah/yq@master
+        with:
+          cmd: yq ".all.children.production.children | keys" ops/inventories/production.yml -o=json | tr '\n' ' '
+      - id: set-matrix
```

Use consistent formatting; in this case, snake case.

Also, this step does not actually set any matrix.

```suggestion
      - id: get_instances_matrix_list
```